src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs

```diff
@@ -332,6 +371,22 @@ protected SourceRepository CreateRepositoryFromSource(string source)
                 {
                     source = outputPath;
                 }
+                else if (exists == false)
```

Comment:
The method `TryTranslatePSPath` needs a bool? type. And, it seems that a bool? type needs an explicit comparison since it can be null.